test/parallel/test-async-wrap-trigger-id.js

```diff
@@ -11,18 +11,18 @@ let triggerAsyncId1;
 process.nextTick(() => {
   process.nextTick(() => {
     triggerAsyncId1 = triggerAsyncId();
+    // Async resources having different causal ancestry
+    // should have different triggerAsyncIds
     assert.notStrictEqual(
```

Comment:
Nit: I think this will now fit in one line, wouldn't it? (And same in the other place)

Reply:
Hmm……because it cannot pass eslint's check, you can see this if I merge them into one line:
![eslinterror](https://user-images.githubusercontent.com/40081831/45911203-f443c080-be42-11e8-9dc2-0deb2c1783b5.PNG)
On the other hand, I strictly follow the rule that in each line there're 80 characters at most :)

Reply:
I think @lundibundi meant that the `assert.strictEqual()` could fit on a single line, not the comment.